<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TlsSessionKeyFilesFrame</class>
 <widget class="QFrame" name="TlsSessionKeyFilesFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TLS Session Key Files</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListView" name="sessionKeyFilesView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addSessionKeyFileButton">
          <property name="text">
           <string>Add new keyfileâ€¦</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteSessionKeyFileButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove key</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>

     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
